### ISSUE TITLE
incorrect header check

### ISSUE THREAD

**Issue by u9g** [willcrichton/flowistry#75]
# Problem
Happens after ctrl+shift+p then Flowistry set focus mark is done

# Logs

**OS:** linux (5.17.0-1021-oem)
**VSCode:** 1.80.0
**Error message**
```
Error: incorrect header check
```

**Full log:** https://paste.rs/AbLXf

**Comment by willcrichton** [comment]
Hi this should be fixed in 0.5.37. Can you confirm this is fixed?

**Comment by u9g** [comment]
> Hi this should be fixed in 0.5.37. Can you confirm this is fixed?

Just tried again, looks like a new problem :)

```
Running command: 	cargo +nightly-2023-04-12 flowistry spans /home/u9g/code/7-28-23/trustfall/trustfall_wasm/src/lib.rs
error: unexpected argument '--cdylib' found

Usage: cargo check --target-dir <DIRECTORY> --quiet --package [<SPEC>]

For more information, try '--help'.
```

Would you prefer me to open a new issue for this?

**Comment by willcrichton** [comment]
No need, I was able to repro. I'll debug tonight and close this issue once it's fixed.

**Comment by willcrichton** [comment]
Found and fixed the issue, will be solved when 0.5.38 is published tonight. Thanks for the reports!
